### PR TITLE
[Tizen Studio] 4.1.1 release notes

### PR DESCRIPTION
**Pull request by manishtr**
Signed-off-by: manish.r <manish.r@samsung.com>

### Change Description ###

[Tizen Studio] 4.1.1 release notes

### CODE REVIEW

**Review by sangwook96 — APPROVED** [review]
Looks good to me!!!

File: docs/application/tizen-studio/release-notes/4-1-1-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Tizen Studio 4.1.1 Release Notes
+
+- Release Date: June 21, 2021
```
Comment (dkson95) [inline docs/application/tizen-studio/release-notes/4-1-1-release-notes.md]: TBD

File: docs/application/toc_all.md [hunk 2]
```diff
@@ -1179,6 +1179,7 @@
 ### [Overview](/application/tizen-studio/rt-ide/overview.md)
 
 ## Release Notes
+### [4.1.1 (June 21, 2021)](/application/tizen-studio/release-notes/4-1-1-release-notes.md)
```
Comment (dkson95) [inline docs/application/toc_all.md]: TBD for date

**Review by sangwook96 — APPROVED** [review]
Looks good to me!!

**Review by mijin85cho — COMMENTED** [review]
@annie-samsung Hello.
Would you please give this PR high priority? Tizen Studio 4.1.1 will be published within this week. 
(Approximately this Thursday)

So we have to merge this update to live branch within this Thursday.

The points where should be checked are only fixed bug line and the release date, comparing to previous release note.
So I think this review can be done in short time.

- Compatibility issue with MacOS Big Sur has been fixed.

File: docs/application/tizen-studio/release-notes/4-1-1-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+- IDE
+  - MacOS Big Sur support has been added.
+
+
+### Deprecated
+   - Tizen RT IDE has been deprecated from Tizen Studio 4.1.
+   - Tizen CLI does not support C# projects from Tizen 5.5 onwards.
+   - Bridge Networking support in emulators will be deprecated in future Tizen Studio releases.
+
```
[the remark this answers is not in the document]
Comment (dkson95) [reply docs/application/tizen-studio/release-notes/4-1-1-release-notes.md:17]: Agreed 👍 @manishtr would you remove this?

[the remark this answers is not in the document]
Comment (manishtr) [reply docs/application/tizen-studio/release-notes/4-1-1-release-notes.md]: done, thanks for your review

**Review by sangwook96 — COMMENTED** [review]
@manishtr 
Hello,
That is not deprecated yet.
- Bridge Networking support in emulators will be deprecated in future Tizen Studio releases.

**Comment by manishtr** [comment]
> @manishtr
> Hello,
> That is not deprecated yet.
> 
> * Bridge Networking support in emulators will be deprecated in future Tizen Studio releases.

Added it back Mr Lee, thanks for your review

**Review by dkson95 — APPROVED** [review]

File: docs/application/tizen-studio/release-notes/4-1-1-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# Tizen Studio 4.1.1 Release Notes
+
+- Release Date: June 22, 2021
+
+## IDE and Tools
+
+### Fixed Bugs
+
+- IDE
+  - Compatibility issue with MacOS Big Sur has been fixed.
```
Comment (annie-samsung) [inline docs/application/tizen-studio/release-notes/4-1-1-release-notes.md:10]:
```suggestion
  - Compatibility issue with macOS Big Sur has been fixed.
```

**Review by annie-samsung — COMMENTED** [review]
All other content is from previous release notes hence no comments for that part.

**Review by annie-samsung — APPROVED** [review]
Looks good

**Comment by annie-samsung** [comment]
@manishtr, @mijin85cho, I will merge this PR when the release is confirmed.

@manishtr, you might have to update the date in the release notes based on the release timeline.

**Comment by manishtr** [comment]
> @manishtr, @mijin85cho, I will merge this PR when the release is confirmed.
> 
> @manishtr, you might have to update the date in the release notes based on the release timeline.

Sure Annie, thanks, for now i will update the date as tomorrows date, incase the date changes again then i will update it also.

File: docs/application/tizen-studio/release-notes/4-1-1-release-notes.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
+# Tizen Studio 4.1.1 Release Notes
+
+- Release Date: June 24, 2021
```
Comment (annie-samsung) [inline docs/application/tizen-studio/release-notes/4-1-1-release-notes.md:3]:
```suggestion
- Release Date: Jun 24, 2021
```

**Review by annie-samsung — APPROVED** [review]
looks good

**Pull request merged**